src/chi/components/picker/picker.scss

```diff
@@ -17,19 +17,39 @@ $sizes: (
 .chi-label,
 label,
 legend {
-  & + .m-picker {
+  & + .chi-picker__item {
     margin-top: 0.25rem;
   }
 }
 
-.m-picker {
+.chi-picker__item {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
   input {
     &[type='checkbox'],
     &[type='radio'] {
-      &.a-picker {
+      &.chi-picker {
+        left: -9999px;
+        position: absolute;
+
         & + label {
+          align-items: center;
+          background-color: $text-white;
+          border: solid 0.0625rem set-color(grey, 30);
+          box-shadow: map-get($shadow, 1);
+          display: flex;
+          font-size: $text;
+          font-weight: normal;
+          justify-content: space-between;
+          margin-bottom: 0.5rem;
+          padding: 1rem - $border;
+          transition: all 0.2s;
+          width: 100%;
+
           span {
```

Comment:
```suggestion
          span {
            border: 2px solid set-color(grey, 50);
            color: set-color(grey, 50);
            height: 1rem;
            position: relative;
            width: 1rem;
            color: set-color(grey, 50);
            height: 1rem;
            position: relative;
            width: 1rem;

            &.chi-picker__checkbox {
              &::before {
                border: 2px solid set-color(grey, 50);
                border-right: 0;
                border-top: 0;
                content: '';
                height: 0.3125rem;
                left: 0.0625rem;
                position: absolute;
                top: 0.125rem;
                transform: rotate(-45deg) scale(0);
                transition-duration: 0.2s;
                width: 0.625rem;
              }
            }

            &.chi-picker__radio {
              border-radius: 0.5rem;

              &::before {
                background-color: set-color(grey, 50);
                border-radius: 0.25rem;
                content: '';
                height: 0.5rem;
                left: calc(50%-0.25rem);
                position: absolute;
                top: calc(50%-0.25rem);
                transform: scale(0);
                transition-duration: 0.2s;
                width: 0.5rem;
              }
            }
          }
```

Reply:
@janibolkvadze good catch! But I didn't do it because there can be cases where there are <span> inside <label> and are not necessarily a checkbox or a radio, it can be plain text like in our size examples.

Reply:
Makes sense 👍